experimental/CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ All notable changes to experimental packages in this project will be documented
 
 ### :bug: (Bug Fix)
 
+* fix(instrumentation-http): improve metrics test stability [#3242](https://github.com/open-telemetry/opentelemetry-js/pull/3242) @pichlermarc
```

Comment:
I think `fix` is the wrong subject. I would use `ci` and move it to the internal section

Reply:
Done in [bb1f218](https://github.com/open-telemetry/opentelemetry-js/pull/3242/commits/bb1f218bd5ca46f9c7bf4f1005df2668572289da). :slightly_smiling_face: